.github/release-please.yml

```diff
@@ -10,3 +10,5 @@ branches:
     handleGHRelease: true
     releaseType: java-backport
     branch: 3.7.x
+extraFiles:
+  - src/main/java/com/google/cloud/logging/Instrumentation.java
```

Comment:
this is per-configuration, so you'll want to add this to the other branch configurations (unless the file does not exist there).

Reply:
Oh, you mean for LTS branch? If for LTS - we dont have to do it there since instrumentation is internal feature and I am OK with previous logic which supposed to work there